### PR TITLE
fix(python): stop processes when force stop

### PR DESCRIPTION
**Pull request by roggervalf**

### CODE REVIEW

File: python/bullmq/worker.py [hunk 1]
```diff
@@ -220,12 +220,20 @@ async def close(self, force: bool = False):
...
             self.forceClosing = True
+            self.cancelProcessing()
 
         self.closing = True
 
         await self.blockingRedisConnection.close()
         await self.redisConnection.close()
 
+    def cancelProcessing(self):
+        print('hola')
```
Comment (manast) [inline python/bullmq/worker.py:231]: Seems to be some debug code left here.

File: python/bullmq/worker.py [hunk 2]
```diff
@@ -220,12 +220,20 @@ async def close(self, force: bool = False):
...
+        for job in self.processing:
+            print('1u')
+            hola = job.cancel()
```
Comment (manast) [inline python/bullmq/worker.py:234]: What is the effect of job.cancel()?

Comment (roggervalf) [reply python/bullmq/worker.py]: it will stop the execution of that job as in https://www.pythontutorial.net/python-concurrency/python-cancel-tasks/

Comment (manast) [reply python/bullmq/worker.py]: Interesting.

**Review by manast — APPROVED** [review]

**Pull request merged**